docs/tutorials/shor.ipynb

```diff
@@ -171,9 +171,9 @@
     "    Args:\n",
     "        n: Modulus of the multiplicative group.\n",
     "    \"\"\"\n",
-    "    assert n > 2\n",
-    "    group = [1, 2]\n",
```

Comment:
[no change needed] I suspect this was built using the common assumption for Shor's algorithm that `n` is an odd number, but the new code is more generally correct. The alternative would be to add a comment explaining the Shor-based optimization being done here.